### ISSUE TITLE
chinese greenhouse

### ISSUE THREAD

**Issue by mikkokotila** [eka-foundation/home#46]
This might be relevant: 

https://www.lowtechmagazine.com/2015/12/reinventing-the-greenhouse.html

Will do a separate post related with aquaphonics, which I think might be a critically important solution for the future of these communities (as water runs out).

**Comment by thepoeticdream** [comment]
Yes, it's relevant and the fundamentals agree with [Solar Green Houses for the Trans-Himalayas](http://www.solaripedia.com/files/1086.pdf) The limited building material supply chain in Ladakh constrains what could be achieved vs. the passive solar greenhouses in China where more modern efficicient materials are readily available.

Aquaponics is very much something that could be the future in the evolution of Zangskar agruiculture but it's much later down the road requiring better infrastructure.

**Comment by mikkokotila** [comment]
> Aquaponics is very much something that could be the future in the evolution of Zangskar agruiculture but it's much later down the road requiring better infrastructure.

What kind of infrastructure do you have in mind?

**Comment by thepoeticdream** [comment]
Electrical infrastructure mainly.  Aquaponics is water pumps, water filtration, water aeration, LED grow lights which requires electricity.  Needs to be reliable electricity or all the fish can die in the aquaponics setup because water aeration not happening from power cut etc.  Try to scale aquaponics production sufficient for the region and that becomes a lot of electricity required.  Feel the route that Zanskar needs to evolve to long term is a solar economy.  If the electrical transmission line to the region gets built with [Ladakh will soon be home to world's largest solar plant ](https://timesofindia.indiatimes.com/india/ladakh-will-soon-be-home-to-worlds-largest-solar-plant/articleshow/67507422.cms)  Then perhaps Zanskar could be an exporter of solar PV electricity elsewhere via that transmission line and become a driver for the local economy.

**Comment by mikkokotila** [comment]
@thepoeticdream thanks for clarifying. Very useful to understand this clearly.
